src/algolia-projects.json

```diff
@@ -1,4 +1,16 @@
 [{
+    "name": "Atom.io",
```

Comment:
Do we have a consensus on this :)? I like Atom npm search, but I also feel that it doesn't imply a lot about how helpful the tool itself is :/